File: cyberdevs.py
class CyberDevs:
    def __init__(self):
        self.name = "CyberDevs"
        self.location = "Islamabad, Pakistan"
        self.specialties = ["Android Development", "Game Creation", "Cybersecurity"]

    def greet(self):
        print(f"🌌 Welcome to the official repository of {self.name}!")
        print(f"📍 Based in {self.location}")
        print("⚡ Our Specialties:")
        for spec in self.specialties:
            print(f"  - {spec}")

if __name__ == "__main__":
    devs = CyberDevs()
    devs.greet()

File: cyberdevs.cs
using System;
using System.Collections.Generic;

namespace CyberDevs
{
    class Program
    {
        static void Main(string[] args)
        {
            var name = "CyberDevs";
            var location = "Islamabad, Pakistan";
            var specialties = new List<string> { "Android Development", "Game Creation", "Cybersecurity" };

            Console.WriteLine($"🌌 Welcome to the official repository of {name}!");
            Console.WriteLine($"📍 Based in {location}");
            Console.WriteLine("⚡ Our Specialties:");
            foreach (var spec in specialties)
            {
                Console.WriteLine($"  - {spec}");
            }
        }
    }
}
